src/ui/utils.js

```diff
@@ -478,3 +493,31 @@ function getTableCellAtPosition( position ) {
 
 	return isTableCellSelected ? position.nodeAfter : findAncestor( 'tableCell', position );
 }
+
+// Returns bounding rect for list of rects.
+//
+// @param {Array.<module:utils/dom/rect~Rect>|Array.<*>} list List of `Rect`s or any list to map by `mapFn`.
+// @param {Function} mapFn Mapping function for list elements.
+// @returns {module:utils/dom/rect~Rect}
+function createBoundingRect( list, mapFn ) {
+	const rectData = {
```

Comment:
that's minor, but could you refactor this to just `rect` - I can see that other code for rects is using this pattern for calculations and it will be easier to search for rect calculations in the whole codebase.

Reply:
it would conflict with `rect` variable in loop